### PR TITLE
Update the temperature reading in outcar.py

### PR DESCRIPTION
**Pull request by jyang2009**
Changed the reading of temperature in OUTCAR. When vasp outputs too high temperature, the parser can now read it and throw a warning when the output is "******"

### CODE REVIEW

**Comment by pmrv** [comment]
The `format_black` fails with

```
Run actions/checkout@v2
Error: Input required and not supplied: token
```

@liamhuber You know that this is about?

**Comment by jan-janssen** [comment]
> @liamhuber You know that this is about?

The pull request is coming from an external fork, so the internal GitHub Actions do not work.

**Comment by niklassiemer** [comment]
That is a PR from a fork. The Fork does not get any credentials from the original repo. Might be that this is related? On the other hand, the action should just update the branch from which the PR is made...

File: pyiron_atomistics/vasp/outcar.py [hunk 1]
```diff
@@ -643,7 +643,12 @@ def get_temperatures(filename="OUTCAR", lines=None):
             for j in trigger_indices:
                 line = lines[j].strip()
                 line = _clean_line(line)
-                temperatures.append(float(line.split()[-2]))
+                output_string = line.split('temperature')[-1].split()[0]
+                try:
+                    temperatures.append(float(output_string))
+                except ValueError:
+                    warnings.warn(f"Temperature too high. Vasp output: {line}")
```
Comment (pmrv) [inline pyiron_atomistics/vasp/outcar.py]:
```suggestion
                    warnings.warn(f"Temperature too high. Vasp output: {line}")
```

**Comment by liamhuber** [comment]
> On the other hand, the action should just update the branch from which the PR is made...

I'm not sure that works either, as then out bot needs write access to the contributor's repo -- the branch in question here is showing as `jyang2009:main`. I guess it might have this permission, but I don't think we can guarantee it?

**Comment by pmrv** [comment]
rebased in #1337

**Pull request closed without merging**